internal/worker/queue_test.go

```diff
@@ -50,42 +58,57 @@ func TestNewQueue(t *testing.T) {
 		if !errors.Is(err, w.err) {
 			return errors.Errorf("got error = %v, want %v", err, w.err)
 		}
-		if !reflect.DeepEqual(got, w.want) {
+		opts := []cmp.Option{
+			cmp.Comparer(func(want, got Queue) bool {
+				return want != nil && got != nil
```

Comment:
do we need to check the initialized value of `Queue`?

Reply:
I think you can refer to this implementation.
https://github.com/vdaas/vald/blob/test/internal/worker/internal/worker/worker_test.go#L70